src/app/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -32,9 +32,16 @@ const LinkContents = ({ item, isInline }) => {
     return content;
   }
 
-  const type = isPhotoGallery
-    ? 'photogallery'
-    : pathOr(null, ['media', 'format'], item);
+  const getAnnouncedType = () => {
+    if (isPhotoGallery) {
+      return 'photogallery';
+    }
+
+    const mediaType = pathOr(null, ['media', 'format'], item);
+    return mediaType === 'audio' ? 'listen' : mediaType;
+  };
+
+  const type = getAnnouncedType();
```

Comment:
`type` on L44 could be `null` if `mediaType` on L40 is `null`. Is that expected?

Reply:
Yes that is expected. Media type should not be null at all because non-media/photogallery items are filtered before this stage. In the event of incomplete data from ARES, type will be null and no type will be announced. So that's expected.

Reply:
Thanks for explaining, Gerald.